src/core/lib/channel/channel_args.c

```diff
@@ -243,7 +243,7 @@ grpc_channel_args *grpc_channel_args_set_compression_algorithm(
   GPR_ASSERT(algorithm < GRPC_COMPRESS_ALGORITHMS_COUNT);
   grpc_arg tmp;
   tmp.type = GRPC_ARG_INTEGER;
```

Comment:
Similarly, this could be changed to use `grpc_channel_arg_integer_create()`.

Same thing throughout this file.